docs/coordinates/skycoord.rst

```diff
@@ -332,7 +332,13 @@ To store arrays of coordinates in a |SkyCoord| object::
 
 In addition to vectorized transformations, you can do the usual array slicing,
 dicing, and selection using the same methods and attributes that you use for
-`~numpy.ndarray` instances::
+`~numpy.ndarray` instances.  Similarly, on ``numpy`` version 1.17 or later,
+corresponding functions as well as others that affect the shape, such as
+`~numpy.at_least1d` and `~numpy.rollaxis`, work as expected.
+(The relevant functions have to be explicitly enabled; let us know if a
```

Comment:
```suggestion
(The relevant functions have to be explicitly enabled in ``astropy``; let us know if a
```
To clarify that the user doesn't need to enable anything: something has to be changed in the astropy code itself!

Reply:
Good idea! I'll do it in all the places where I added this... (and fix the docs build at the same time)